src/utils/dts.ts

```diff
@@ -56,8 +58,7 @@ export function extractDeclarations(
     const dtsFilename = filename.replace(/\.(m|c)?(ts|js)x?$/, ".d.$1ts");
     let contents = vfs.get(dtsFilename) || "";
     if (opts?.addRelativeDeclarationExtensions) {
-      const ext =
-        filename.match(/\.(m|c)?(ts|js)$/)?.[0].replace(/ts$/, "js") || ".js";
+      const ext = filename.match(EXT_RE)?.[0].replace(/ts$/, "js") || ".js";
```

Comment:
We maintain other regexes inline (and in modern v8 that has inline-cache there is no perf penalty of keeping regex inline)

Reply:
I was only thinking because we use it twice. If you don’t think it’s significant I’ll inline it in both.